openshift_pipelines/installing-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="installing-pipelines"]
+= Installing OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: installing-pipelines
+
+toc::[]
+
+// Installing Openshift Pipelines Operator
+[id="installing-pipelines-operator_{context}"]
+== Installing the {op-title} Operator
+
+You can install the {op-title} Operator in the host cluster by following the {product-title} instructions specified in xref:../operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster].
+
+// Deleting Openshift Pipelines Operator
+[id="deleting-openshift-pipelines-operator_{context}"]
+== Deleting the {op-title} Operator
```

Comment:
@nikhil-thomas could you confirm if deleting the operator would delete tekton controllers?

Reply:
@nikhil-thomas - Could you please reply to the comment above. Thanks

Reply:
Added a new section to Uninstall the Pipelines Operator